docs/api/covidcast-signals/hhs.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Department of Health & Human Services
+parent: Data Sources and Signals
+grand_parent: COVIDcast Epidata API
+---
+
+# SOURCE NAME
+{: .no_toc}
+
+* **Source name:** `hhs`
+* **First issued:** TBD
+* **Number of data revisions since 19 May 2020:** 0
+* **Date of last change:** Never
+* **Available for:** county, hrr, msa, state (see [geography coding docs](../covidcast_geography.md))
+* **Time type:** day (see [date format docs](../covidcast_times.md))
+* **License:** [Open Data Commons Open Database License (ODbL)](https://opendatacommons.org/licenses/odbl/1-0/)
+
+The U.S. Department of Health & Human Services (HHS) publishes several
+datasets on patient impact and hospital capacity. Two of these
+datasets are mirrored in Epidata at the following endpoints:
+
+* [COVID-19 Hospitalization: States](../covid_hosp.md) - daily resolution, state aggregates
+* [COVID-19 Hospitalization: Facilities](../covid_hosp_facilities.md) - weekly resolution, individual hospitals
+
+Both datasets contain dozens of columns that break down hospital
+resource usage in different ways.
+
+This indicator makes available several commonly-used combinations of
+those columns, aggregated geographically. In particular, we include
+the sum of all adult and pediatric COVID-19 hospital admissions. This
+sum is used as the "ground truth" for hospitalizations by the (Reich
+lab COVID-19 forecast
+hub](https://github.com/reichlab/covid19-forecast-hub/blob/master/data-processed/README.md#hospitalizations].
+
+
+| Signal | Geography | Resolution | Description |
+| --- | --- |
+| `confirmed_admissions_1d` | state | 1 day | Sum of adult and pediatric confirmed COVID-19 hospital admissions occuring each day. |
+| `sum_confirmed_suspected_admissions_1d` | state | 1 day | Sum of adult and pediatric confirmed and suspected COVID-19 hospital admissions occuring each day. |
+| `confirmed_admissions_7d` | county, hrr, msa, state | 7 days | Sum of adult and pediatric confirmed COVID-19 hospital admissions occuring over a span of 7 days. |
+| `sum_confirmed_suspected_admissions_7d` | county, hrr, msa, state | 7 days | Sum of adult and pediatric confirmed and suspected COVID-19 hospital admissions occuring over a span of 7 days. |
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+## Estimation
+
+### Statewise, daily resolution
+
+Statewise daily resolution signals use the following four columns from
+the HHS state timeseries dataset:
```

Comment:
Make "timeseries" either "time-series" or "time series".

Reply:
I happen to agree, but alas "timeseries" is the spelling established by the source:
https://healthdata.gov/dataset/covid-19-reported-patient-impact-and-hospital-capacity-state-timeseries

Reply:
Ah in that case, ignore this suggestion!